mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/AuthenticationServiceImpl.java

```diff
@@ -257,7 +301,7 @@ private Boolean doKycAuthentication(KycAuthRequestDto kycAuthRequestDto,Identity
         }
 
         if (kycAuthRequestDto.getPin() != null) {
-            authStatus = kycAuthRequestDto.getPin().equals(identityData.getPin());
+            authStatus = kycAuthRequestDto.getPin().equals(HelperUtil.getIdentityDataValue(identityData,"pin","eng"));
```

Comment:
lets use already defined "defaultLanguage" variable.

Reply:
used the default language